modules/framework/galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/ITestMethodResult.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package dev.galasa.framework;
+
+public interface ITestMethodResult {
+    String getMethodName();
+    boolean isPassed();
+    boolean isFailed();
+    Throwable getThrowable();
```

Comment:
Should `getThrowable` instead be `getFailureReason` like the interface in the story mentions?

Reply:
Thanks Eamonn, addressed in https://github.com/galasa-dev/galasa/pull/384/commits/e911df37778a48589842d40fb0309320cf0513dc